pmd-cataloger/src/main/java/sfdc/sfdx/scanner/pmd/ScannerPmdException.java

```diff
@@ -0,0 +1,17 @@
+package sfdc.sfdx.scanner.pmd;
+
+/**
+ * Custom exceptions for PMD-based scanner to handle errors in a special way
+ */
+public class ScannerPmdException extends RuntimeException {
```

Comment:
For now, I've been using custom exceptions for error handling so that debugging is easier. Can switch to error codes to align with rest of the convention followed.